File: docs/snippets/export_compiled_model.py
# Copyright (C) 2018-2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
#

device = "CPU"

#! [export_compiled_model]

import openvino as ov

ov.Core().compile_model(device, modelPath, properties).export_model(compiled_blob)

#! [export_compiled_model]

File: docs/snippets/export_compiled_model.cpp
#include <fstream>
#include <openvino/runtime/core.hpp>

int main() {

//! [export_compiled_model]
ov::Core core;
ov::CompiledModel model = core.compile_model("modelPath", "deviceName");
std::fstream stream("compiled_model.blob");
model.export_model(stream);
//!  [export_compiled_model]

return 0;
}

